esiclient/tests/functional/utils.py

```diff
@@ -0,0 +1,68 @@
+#    Copyright (c) 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+from tempest.lib.common.utils import data_utils
+from tempest.lib import exceptions
+
+
+def node_create(client, driver='fake-hardware', name=None, params='',
+                fail_ok=False, merge_stderr=False):
+
+    if not name:
+        name = data_utils.rand_name('baremetal')
+
+    output = client.openstack('baremetal node create '
+                              '--driver {0} --name {1} {2} -f json'
+                              .format(driver, name, params),
+                              ' ', ' ', fail_ok, merge_stderr)
+
+    return json.loads(output)
+
+
+def node_delete(client, identifier, ignore_exceptions=False,
```

Comment:
`ignore_exceptions` here appears to serve the same functionality as `fail_ok`. This is my bad though; I probably should have documented the behavior of `fail_ok` better. See: https://opendev.org/openstack/tempest/src/branch/master/tempest/lib/cli/base.py

Reply:
I agree. I know I accidentally named it so after peeking into the Ironic functional tests, but for consistency's sake Ill change it